pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -493,3 +508,37 @@ func (p *Provisioner) populateDomainAndPort(sc *storagev1.StorageClass) error {
 
 	return nil
 }
+
+// Check for quota limit in OBC and set accordingly
+func (p Provisioner) SetObjectBucketClaimQuota(options *apibkt.BucketOptions) error {
+	maxObjects := getMaxObject(options)
+	maxSize := getMaxSize(options)
+	var err error
+
+	if maxObjects != "" {
+		_, err = cephObject.SetQuotaUserObjectMax(p.objectContext, p.cephUserName, maxObjects)
+		if err != nil {
+			logger.Errorf("Setting MaxObject failed %v", err)
+			//p.deleteOBCResource(p.bucketName)
```

Comment:
We can remove this comment since the caller deletes it?

Reply:
Oops my bad